app/src/main/res/layout/dialog_cancel_relation.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:orientation="vertical"
+              android:background="#f9f6e5"
```

Comment:
Define this color in **colors.xml** file and give the reference of that color here.